architecture/additional_concepts/networking.adoc

```diff
@@ -22,6 +22,63 @@ recommend that you create a
 link:../core_objects/kubernetes_model.html#service[service], then interact with
 the service.
 
+== OpenShift DNS
+
+If you are running multiple
+link:../core_objects/kubernetes_model.html#service[services], such as frontend
+and backend services for use with multiple pods, in order for the frontend pods
+to communicate with the backend services, environment variables are created for
+username, service-ip, etc. If the service is deleted and recreated, a new IP
+address can be assigned to the service, and requires the frontend pods to be
+recreated in order to pick up the updated values for the service IP environment
+variable. Additionally, the backend service has to be created before any of the
+frontend pods to ensure that the service IP is generated properly and that it
+can be provide to the frontend pods as an environment variable.
+
+For this reason, OpenShift has an inbuilt DNS so that the services can be
+reached by the service DNS as well as the service IP/port. OpenShift supports
+split DNS by running a SkyDNS on the master that answers DNS queries for
```

Comment:
Maybe:

s/a SkyDNS/SkyDNS/

And possibly embed link to https://github.com/skynetservices/skydns.